src/main/java/org/testng/internal/Invoker.java

```diff
@@ -434,14 +434,21 @@ private boolean confInvocationPassed(ITestNGMethod method, ITestNGMethod current
           result = !classConfigurationFailed(cls);
         } else {
           synchronized (m_classInvocationResults) {
-            Set<Object> set = m_classInvocationResults.get(cls);
-            //If the set was null, it means that the test method is residing in a child class
+            Set<Object> set = null;
```

Comment:
Does this change have anything to do with the removal of `@ExpectedExceptions`?

Reply:
@cbeust - No this is not related to this changeset. But this is basically fixing a bug that got introduced as part of 879c33910f620ded24657e7794cb6f5baf812dd7

But what I am not able to understand is, how did the build pass previously and how come it fails now.